domain/model/service/service.go

```diff
@@ -464,6 +474,21 @@ func ModelTypeForCloud(
 	return coremodel.IAAS, nil
 }
 
+// ListModelSummariesForUser returns a slice of model summaries for a given
+// user. If no models are found an empty slice is returned.
+func (s *Service) ListModelSummariesForUser(ctx context.Context, userName string) ([]model.ModelSummary, error) {
```

Comment:
Should we being using a user uuid here to be exact ?

Reply:
The user can supply the username with the `--user` flag to this function, so imo it should really be the user name, otherwise we would have to do a separate call to get the user.

Do you think we should return a `UserNotFound` error if the current user cannot be found? Currently we do not

Reply:
Yeah I think we should be exact and well constrained where we can.